lib/mdot/response.rb

```diff
@@ -30,11 +32,8 @@ def determine_eligibility
       eligibility = MDOT::Eligibility.new
 
       supplies.each do |supply|
-        group = supply.product_group.split.last.to_sym
-
-        if eligibility.attributes.key?(group)
-          eligibility.attributes = { group => supply.next_availability_date <= Time.zone.today }
-        end
+        group = supply.product_group.downcase.to_sym
```

Comment:
Based on what it was previously, does not doing a split on product_group and grabbing the last element have the potential to cause issues? Not familiar with what a typical value would be so just trying to understand.

Reply:
So the product_groups were changed a little bit. Prior to this they were like "hearing aid batteries" or "hearing aid accessories" so we would have to pull out the last word and symbolize it, but now they are like "BATTERIES" so doing the split is no longer necessary.